examples/cmd/vppagent-nsc/nsc.go

```diff
@@ -80,15 +80,16 @@ func main() {
 		logrus.Fatalf("Unable to create the backend %v", err)
 	}
 
-	var outgoingConnection *connection.Connection
-	outgoingConnection, err = client.Connect("if1", "mem", "Primary interface")
+	err = clientList.Connect("if1", "mem", "Primary interface")
 	if err != nil {
 		logrus.Fatalf("Unable to connect %v", err)
 	}
 
-	err = backend.Connect(outgoingConnection)
-	if err != nil {
-		logrus.Fatalf("Unable to connect %v", err)
+	for _, conn := range clientList.GetConnections() {
```

Comment:
Again, I don't get it why we need all this? I would expect we have a single connection.

Reply:
ditto

Reply:
Your proposal makes sense, since sdk.client.Connect() returns a Connection, sdk.clientList.Connect() should return a slice of Connection.

I'll update my patch this way.